metrics/prometheus/prometheus.go

```diff
@@ -952,6 +959,16 @@ func (m *Metrics) RecordRequestPrivacy(privacy metrics.PrivacyLabels) {
 	}
 }
 
+func (m *Metrics) RecordAdapterBuyerUIDScrubbed(adapterName openrtb_ext.BidderName) {
+	if m.metricsDisabled.AdapterGDPRRequestBlocked {
+		return
```

Comment:
Is it possible to add a test to cover this return?

Reply:
Added coverage and fixed the incorrect condition here.